softwerkskammer/teststubs/nodemailerTransportForTests.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  sendMail: function (transportobject, callback) {
+    if (!transportobject.to && (!transportobject.bcc || transportobject.bcc.length === 0)) {
```

Comment:
Wollen wir hier nicht das gleiche Verhalten haben wie bei der Testpersistence? Also dass man in den Tests gezwungen wird, die Funktion zu mocken. Das fände ich am saubersten.

Das sähe dann so aus:
`function () {throw new Error('Nodemailer Transport for tests - function sendMail not implemented'); }`